src/main/cpp/t05_palindrom.cpp

```diff
@@ -18,10 +18,24 @@
 
 #include "t05_palindrom.h"
 #include <iostream>
+#include<string>
 
 
 using namespace std;
 
 int t05_palindrom() {
 
+  string str;
+  int k;
+  bool flag=flag;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Redundant assignment of 'flag' to itself.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4513001)